keycloak/keycloak.go

```diff
@@ -111,6 +112,24 @@ func (k *Keycloak) GetUser(ctx context.Context, userID string) (*User, error) {
 	user.StripeCancelationTime, _ = strconv.ParseInt(safeGetAttr(kcuser, "stripeCancelationTime"), 10, 0)
 	user.StripeETag = safeGetAttr(kcuser, "stripeCancelationTime")
 
+	kuserlogins, err := k.client.GetUserFederatedIdentities(ctx, token.AccessToken, k.env.KeycloakRealm, *kcuser.ID)
+	
```

Comment:
nit: it's idiomatic to not leave an empty line between a stanza and it's error handling code

```suggestion
```

Reply:
Good to know, will fix